--- src/scripts/Makefile.in.orig	Thu Nov  1 17:52:19 2001
+++ src/scripts/Makefile.in	Thu Nov  1 17:53:08 2001
@@ -54,7 +54,7 @@
 	@(v=`cat $(top_srcdir)/VERSION`; \
 	  v="$${v} (`sed 's|/|-|g' $(top_srcdir)/date-stamp`)"; \
 	  y=`cut -d/ -f1 < $(top_srcdir)/date-stamp`; \
-	  sed "s@R_HOME_DIR=.*@R_HOME_DIR=`cd $(top_builddir); pwd`@; \
+	  sed "s@R_HOME_DIR=.*@R_HOME_DIR=$(rhome)@; \
 	       s@REPLACE_BY_VERSION@$${v}@; \
 	       s@REPLACE_BY_YEAR@$${y}@" < R.sh > $@)
 	@chmod +x $@
